.eslintrc.js

```diff
@@ -37,10 +37,16 @@ module.exports = {
       "always",
       { exceptAfterSingleLine: true },
     ],
-    "jsx-a11y/label-has-associated-control": ['error', {
-      "assert":"either"
-    }],
-    "react/static-property-placement": ["error", "static public field"]
+    "jsx-a11y/label-has-associated-control": [
+      "error",
+      {
+        assert: "either",
+      },
+    ],
+    "react/static-property-placement": "off",
+    "class-methods-use-this": "off",
+    "react/no-unused-class-component-methods": "off",
```

Comment:
We have many instances of methods defined on a class but not used in the same file.